server/src/Application/Validation/Validators/Course/UpdateCourseRequestValidator.cs

```diff
@@ -0,0 +1,22 @@
+using FluentValidation;
+using StudyZen.Application.Dtos;
+using StudyZen.Application.Validation.Validators;
+
+namespace StudyZen.Application.Validators;
+
+public class UpdateCourseRequestValidator : BaseValidator<UpdateCourseDto>
+{
+    public UpdateCourseRequestValidator()
+    {
+        RuleFor(course => course.Name)
+        .Must(NullOrNotWhiteSpace)
```

Comment:
Use NotEmpty and add Unless null at the end: https://docs.fluentvalidation.net/en/latest/conditions.html